src/giphyCardAction.ts

```diff
@@ -0,0 +1,14 @@
+import { CardAction } from 'botbuilder';
+import { GiphyAction } from './giphyAction';
+
+export class GiphyCardAction {
+
+    public static deleteCardAction: CardAction = {
+            channelData: {},
```

Comment:
just out of curiosity is `channelData` mandatory? (what source were you able to find with it)